tests/helpers/execute.ts

```diff
@@ -16,13 +15,23 @@ const createProcess = (processPath: string, args: string[] = [], env: Env = unde
 };
 
 const readStream = async (stream: NodeJS.ReadableStream): Promise<string> => {
-  return new Promise<string>((resolve, reject) => {
-    stream.pipe(
-      concatStream((result) => {
-        resolve(result.toString());
-      }).on('error', reject)
-    );
-  });
+  if (stream.readable) {
+    return new Promise<string>((resolve, reject) => {
+      let str = '';
+      stream.setEncoding('utf-8');
+      stream.on('data', (chunk) => {
+        str += chunk;
+      });
+      stream.on('error', (err) => {
+        reject(err);
+      });
+      stream.on('end', () => {
+        resolve(str);
+      });
+    });
+  } else {
+    return Promise.resolve('');
+  }
```

Comment:
it should throw an error. how can you differentiate between empty string as a valid value, and as an error?

Reply:
fixed